docs/metrics.asciidoc

```diff
@@ -60,3 +60,77 @@ This value is normalized by the number of CPU cores and it ranges from 0 to 100%
 
 The Resident Set Size,
 the amount of memory the process occupies in main memory (RAM).
+
+[float]
+[[metric-node.active_handles]]
+=== `node.active_handles`
+
+* *Type:* Long
+* *Format:* Counter
+
+The number of active libuv handles,
+likely held open by currently running I/O operations.
+
+[float]
+[[metric-node.active_requests]]
+=== `node.active_requests`
+
+* *Type:* Long
+* *Format:* Counter
+
+The number of active libuv requests,
+likely waiting for a response to an I/O operation.
+
+[float]
+[[metric-node.cpu.user]]
+=== `node.cpu.user`
+
+* *Type:* Long
+* *Format:* Counter
+
+The number of CPU cycles spent executing application code.
+
+[float]
+[[metric-node.cpu.system]]
+=== `node.cpu.system`
+
+* *Type:* Long
+* *Format:* Counter
+
+The number of CPU cycles spent executing kernel code as a result of application activity.
+
+[float]
+[[metric-node.loop.delay]]
+=== `node.loop.delay`
+
+* *Type:* Float
+* *Format:* Milliseconds
+
+The number of milliseconds of event loop delay.
```

Comment:
I'm not sure how big it is. The `event-loop-delay` module doesn't appear to document that either. 🤔 

Reading the code, I _think_ it's a 5us variance, but it's a bit unclear.

Reply:
I was referring to the 10ms sampling rate that's in the Node.js core version. I can't remember what the sampling rate is in the one @mafintosh created 🤔

Reply:
I'm not using the core one at all currently, since it only exists in Node.js 11+ and doesn't work the same as it has its own built-in histogram layered over it. 🤔